src/drc/api/serializers.py

```diff
@@ -687,7 +689,8 @@ class Meta:
         }
         validators = [
             ObjectInformatieObjectValidator(),
-            InformatieObjectUniqueValidator("object", "informatieobject"),
+            InformatieObjectUniqueValidator("object", "informatieobject", "informatieobject_versie"),
+            InformatieObjectVersionValidator()
```

Comment:
I'd move this validator to the `informatieobject` field, since it's essentially part of it (it applies on the `?versie=` querystring).